pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -87,10 +88,11 @@
   private final Path _inputSegmentDir;
   private final Path _preprocessedOutputDir;
   protected final String _rawTableName;
-  protected final List<PushLocation> _pushLocations;
+  protected List<PushLocation> _pushLocations;
```

Comment:
Remove the member variables that are already in the base segment job (_pushLocations and _rawTableName). Same for SegmentCreationJob.